README.md

```diff
@@ -0,0 +1,58 @@
+# Clear Containers Tests Repository
+
+This repository contains the test suite for the Clear Containers 3.0 project,
+including functional and integration tests with Docker\*, Kubernetes\* and OpenShift\*.
+
+An environment with Clear Containers properly setup is needed in order to execute 
```

Comment:
Change word order:
A properly set up environment with Clear Containers is needed to execute these tests.